hm_pyhelper/hardware_definitions.py

```diff
@@ -528,7 +528,7 @@ def is_raspberry_pi() -> bool:
     'syncrobit-fl1': {
         'FRIENDLY': 'Syncrobit Hotspot',
         'SUPPORTED_MODELS': ['Syncrobit Hotspot'],
-        'CPU_ARCH': 'aarch64',
```

Comment:
aren't they same. I have seen them used interchangeably

Reply:
From architecture standpoint, yes. But since I'm using it for getting hm-miner image names, it conflicts with existing schema, which uses `arm64`.